llvm/test/CodeGen/RISCV/stack-guard-global.ll

```diff
@@ -0,0 +1,50 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -verify-machineinstrs < %s | \
+; RUN: FileCheck %s
+
+target triple = "riscv64-unknown-linux-gnu"
```

Comment:
Convention is to put it in the RUN line

Reply:
sorry, copied this from llvm/test/CodeGen/AArch64/stack-guard-sysreg.ll which did it this way. I'll switch.